src/test/shell/bazel/external_integration_test.sh

```diff
@@ -910,7 +910,7 @@ http_archive(
 )
 EOF
   bazel build @repo//... &> $TEST_log && fail "Expected to fail"
-  expect_log "[Ii]nvalid SHA-256 checksum"
+  expect_log "Invalid SHA256 checksum 'a random string'. The correct checksum is '$(sha256sum repo.zip | head -c 64)'."
```

Comment:
Is the original "[Ii]nvalid SHA-256 checksum" string also being printed? If so, can we merge those messages so there's no duplication? I think the approach taken in https://github.com/bazelbuild/bazel/pull/5996/files might be the correct one.